test/jl/tests.jl

```diff
@@ -142,7 +142,7 @@ function test6()
    println("test 6: Conversion of images from Images.jl to OpenCV Mat")
    filename = joinpath(Pkg.dir("OpenCV"), "./test/images/mandrill.jpg")
    image = imread(filename)
-   dst = Mat(int(rows(image)), int(cols(image)), CV_8UC1)
+   dst = Mat(Int(round(rows(image))), Int(round(cols(image))), CV_8UC1)
    dst = convertToMat(image)
 end
 
```

Comment:
By all means dive in and set up the wrappers! 


I did have a quick look yesterday evening at implementing `cv::Mat<_Tp>` but got errors probably because it was not as straightforward as it first seems.

I do not actually have much time right now to work on this project (definitely not during the week) - but I would like to make sure `OpenCV.jl` continues to work properly and aim always to enhance it in areas where Julia offers unique advantages.